### PR TITLE
CAT-XP6 [MH, PH] AddItemToUserListTest

### PR DESCRIPTION
**Pull request by xsoliah**

### CODE REVIEW

**Comment by 84n4n4** [comment]
ok to test

**Comment by xsoliah** [comment]
please change GIF btw

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/SceneTransmitionBrickTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		String newScene = "testScene2";
+		checkIfBrickAtPositionShowsString(0, R.string.brick_when_started);
+		checkIfBrickAtPositionShowsString(brickPosition, R.string.brick_scene_transition);
+		onScriptList().atPosition(brickPosition).onChildView(withId(R.id.brick_scene_transition_spinner))
+				.perform(click());
+		onView(withText(R.string.brick_variable_spinner_create_new_variable))
+				.perform(click());
+		enterTextOnDialogue(R.id.scene_name_edittext, newScene);
+	}
+}
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/SceneTransmitionBrickTest.java]: missing quite some parts compared to the original robotium test

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/AddItemToUserListTest.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+	@Test
+	public void testAddItemToUserListBrick() {
+		String toInsert = "Element";
+		checkIfBrickAtPositionShowsString(0, R.string.brick_when_started);
+		checkIfBrickAtPositionShowsString(brickPosition, R.string.brick_add_item_to_userlist);
+		createNewVariableOnSpinnerInitial(R.id.add_item_to_userlist_spinner, brickPosition, toInsert);
+		BrickTestUtils.enterValueInFormulaTextFieldOnBrickAtPosition(10, R.id.brick_add_item_to_userlist_edit_text, brickPosition);
+	}
+}
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/AddItemToUserListTest.java]: some tests missing compared to the original robotium test.
please have a look at DeleteItemOfUserListBrickTest

**Pull request closed without merging**